TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -90,45 +91,40 @@ class BenchmarkParser extends Parser {
             }
 
             if ( isValid ) {
-
-                curBenchmarkNamedRouted = BenchmarkMetricRouter[curBenchmarkName][curBenchmarkVariant];
+            	
+                curBench = BenchmarkMetric[BenchmarkMetricRouter[curBenchmarkName][curBenchmarkVariant]];
 
                 // Parse metric values
                 curTestData["metrics"] = [];
-                for ( let i = 0; i < BenchmarkMetric[curBenchmarkNamedRouted]["metrics"].length; i++ ) {
-                    curMetric = BenchmarkMetric[curBenchmarkNamedRouted]["metrics"][i].name;
-                    curSearchString = curItr.value;
+                for ( let i = 0; i < curBench["metrics"].length; i++ ) {
```

Comment:
We are using `curBench["metrics"]` at multiple places. Let's use a var for it instead of using `curBench`. Maybe let's call it `expectedMetricsArray`.